toucan_connectors/common.py

```diff
@@ -404,7 +404,7 @@ def pandas_read_sql(
         # FIXME: We should use here the sqlalchemy.text module to
         # escape characters like % but as a quick fix, we left replace
         query = query.replace('%%', '%')
-        query = re.sub(r'%[^%]+%', r'%\g<0>%', query)
+        query = re.sub(r'%[A-Za-z0-9]+%', r'%\g<0>%', query)
```

Comment:
this is too specific (we cannot do `LIKE '%foo bar%'` for example)
we should just exclude patterns like `%(xxx)s` IMO

Reply:
true, it's better 🙏🏽